chunky/src/java/se/llbit/chunky/block/Head.java

```diff
@@ -1,8 +1,5 @@
 package se.llbit.chunky.block;
```

Comment:
No license notice here (I don't remember, do we care about that?), I'm noticing it because almost all the other files have their copyright information updated, did you setup something to do that automatically?

Reply:
I usually don't care but started updating the headers here... I would like to have something that automates this